src/app/services/alerts.tsx

```diff
@@ -33,5 +34,14 @@ export const useAlertsService: () => AlertsService = (): AlertsService => {
                 dataTestId: "toast-instance-deleted"
             });
         },
+
+        instanceDeleteError(instance) : void {
+            addAlert({
+                title: "Error while deleting the instance",
+                description: `There was an error while marking the Registry instance '${instance.name} for deletion. Please try it again.'`,
+                variant: AlertVariant.danger,
+                dataTestId: "toast-instance-deleted" //TODO Whats that?
```

Comment:
This is supposed to add a "data-testId" attribute to the UI component for the resulting toast alert.  The purpose is for automated UI tests (e.g. selenium tests) to be able to identify the specific toast alert to verify that it appears.

Reply:
Note: I don't know if it works. :)